docs/examples/DataSet/Dataset_Performance.ipynb

```diff
@@ -79,7 +79,7 @@
    "metadata": {},
```

Comment:
>from qcodes.dataset.experiment_container import (Experiment,,
                                                     load_last_experiment,                                                
                                                        new_experiment)

I think these imports are not used anywhere in the notebook.

Reply:
As we mention about the container, I prefer these imports to stay.